recipes/magics-metview/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+export PYTHON="$PYTHON"
```

Comment:
Does this do anything? If it isn't exported you can use this instead:
```suggestion
export PYTHON
```

Reply:
Looks like it does nothing